Polygon/include/CGAL/Polygon_2/Polygon_2_simplicity.h

```diff
@@ -74,6 +74,8 @@ struct Vertex_index {
     explicit Vertex_index(Index_t i): m_i(i) {}
     Index_t as_int() const {return m_i;}
     Vertex_index operator++() {++m_i; return *this; }
+    bool operator==(const Vertex_index& other) const = default;
```

Comment:
`operator==` cannot be `= default`. That is only allowed since C++20. Before C++20, only special member functions (cstor, dstor, copy, assign, and move-*) can be defaulted.
https://en.cppreference.com/w/cpp/language/member_functions#Special_member_functions